hypothesis-python/src/hypothesis/internal/floats.py

```diff
@@ -46,25 +38,30 @@
 # There are two versions of this: the one that uses Numpy to support Python
 # 3.5 and earlier, and the elegant one for new versions.  We use the new
 # one if Numpy is unavailable too, because it's slightly faster in all cases.
-if numpy and not CAN_PACK_HALF_FLOAT:  # pragma: no cover
-
-    def reinterpret_bits(x, from_, to):
-        if from_ == b"!e":
-            arr = numpy.array([x], dtype=">f2")
-            if numpy.isfinite(x) and not numpy.isfinite(arr[0]):
-                quiet_raise(OverflowError("%r too large for float16" % (x,)))
-            buf = arr.tobytes()
-        else:
-            buf = struct_pack(from_, x)
-        if to == b"!e":
-            return float(numpy.frombuffer(buf, dtype=">f2")[0])
-        return struct_unpack(to, buf)[0]
+def reinterpret_bits(x, from_, to):
+    return struct_unpack(to, struct_pack(from_, x))[0]
 
 
-else:
+if not CAN_PACK_HALF_FLOAT:  # pragma: no cover
+    try:
+        import numpy
```

Comment:
Do we not care about non-lazily importing numpy in this case? I guess it only matters until January.

Reply:
Also for Python 3.5, so [until September](https://devguide.python.org/#status-of-python-branches)! 

But basically no, I don't care about this case - pypistats show very few people still using 3.5. A lazy import here would be pretty awful code, impact the floats() check too, and we'd be taking it out in 10 months anyway. I'll add it if someone asks, though.